### ISSUE TITLE
`langword` value is not colorized correctly in doc comments

### ISSUE THREAD

**Issue by DoctorKrolic** [dotnet/roslyn#63885]
**Version Used**:
VS 2022 v17.3.3

**Screenshot**:
![devenv_tnbp9laPEs](https://user-images.githubusercontent.com/70431552/189322549-963eeaf7-ec64-410d-ae4d-ea35e2a9f685.png)

**Expected Behavior**:
`true` inside of a `langword` is colorized the same, as `public`, `bool` and `get` keywords below

**Actual Behavior**:
`true` is not colorized

**Comment by Eagle3386** [comment]
Using latest VS (17.11.5 as of today), I cannot confirm this bug still occurs:
![Image](https://github.com/user-attachments/assets/c1708387-4075-4502-a230-b1a450e78264)

**Comment by CyrusNajmabadi** [comment]
Closing out as not repro.

**Comment by DoctorKrolic** [comment]
@Eagle3386 You don't reproduce the bug probably due to you color theme. With the default dark theme `langword` keywords are not colorized:
![Image](https://github.com/user-attachments/assets/e8bd93f8-5444-43c2-896b-95c19761a267)

**Comment by Eagle3386** [comment]
> [@Eagle3386](https://github.com/Eagle3386) You don't reproduce the bug probably due to you color theme. 
Yup, using `Solarized Dark` from the [Visual Studio Theme Pack](https://marketplace.visualstudio.com/items?itemName=idex.vsthemepack) - but at least, I tried to repo with R# disabled before replying! 😅🥳
